src/ui/connect-dashboard-ui/src/FamilyHubs.RequestForSupport.Core/ApiClients/OrganisationClientService.cs

```diff
@@ -0,0 +1,31 @@
+using System.Text.Json;
+using FamilyHubs.ReferralService.Shared.Dto;
+
+namespace FamilyHubs.RequestForSupport.Core.ApiClients;
+
+public interface IOrganisationClientService
```

Comment:
Is this a general convention to have the interfaces in the same file as the class implementation

Reply:
Generally yes, unless there is going to be multiple implementations of the concrete class. I can see it used elsewhere as well. This could change when we move to clean architecture